src/Pages/Shop/ShopListCol.js

```diff
@@ -1,7 +1,55 @@
-import React, { Component } from "react";
+import React from "react";
+import GoodsList from "../../Components/GoodsList";
+import "./ShopListCol.scss";
+
+class ShopListCol extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      mouseover: true,
+      Goods: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("Data/MockData/Goods.json")
+      .then((response) => response.json())
+      .then((response) => {
+        this.setState({ Goods: response.Goods });
+      });
+  };
+
+  //HoverImgChanger = () => {
+  //  this.setState({
+  //    mouseover: !this.state.mouseover,
+  //  });
+  //};
 
-export default class ShopListCol extends Component {
   render() {
-    return <div></div>;
+    return (
+      <div className="ShopListCol">
+        <aside></aside>
+        <main>
+          <header>
+            <span>BOOTS</span>
+          </header>
+          <GoodsList
+            Goods={this.state.Goods}
+            mouseover={this.state.mouseover}
+          />
+          <nav>
+            <a href="?">WELINE</a>
```

Comment:
```suggestion
            <a>WELINE</a>
```
href 값에 ?를 쓴 이유 너무나도 물음표... ❓

Reply:
주소추가완료하였습니다